### PR TITLE
Tests for the search entity endpoint with an incorrect parameter were added

### PR DESCRIPTION
**Pull request by mpushnikA1QA**
Tests for the search entity endpoint with an incorrect parameter were added

### CODE REVIEW

File: src/util/commonUtils.js [hunk 1]
```diff
@@ -41,11 +41,27 @@ function getRandomNumberInTheRangeFromMinToMax(min, max) {
   return Math.round(rand);
 }
 
+function getRandomStringByLength(length) {
```
Comment (agromyko) [inline src/util/commonUtils.js:44]: we can use generateStringByLength method from framework (randomUtils) as it does the same thing

File: src/constants/apiTextMessage.js [hunk 2]
```diff
@@ -8,13 +8,17 @@ const ALLOWED_FIELDS_FOR_THE_CHANGES_ENDPOINT = (incorrectField, entity) =>
 
 const CHANGES_ENDPOINT_WITHOUT_SINCE_QUERY_WARNING_MESSAGE = "Either since need to be supplied";
 
-const INCORRECT_FIELD_NAME_DOESNT_EXISTS = (incorrectFieldName) =>
+const INCORRECT_FIELD_NAME_DOESNT_EXISTS_FOR_A_GET_METHOD = (incorrectFieldName) =>
   `Field ${incorrectFieldName} do not exist`;
 
+const INCORRECT_FIELD_NAME_DOESNT_EXISTS_FOR_A_POST_METHOD = (incorrectFieldName) =>
+  `Incorrect query:\nProperty "${incorrectFieldName}" doesn't exists`;
```
Comment (agromyko) [inline src/constants/apiTextMessage.js:15]: typo
```suggestion
  `Incorrect query:\nProperty "${incorrectFieldName}" doesn't exist`;
```

Comment (mpushnikA1QA) [reply src/constants/apiTextMessage.js:15]: This is not a typo. This text is now returned by the API

**Review by agromyko — APPROVED** [review]

File: src/util/commonUtils.js (hunk 1 above)
Comment (mpushnikA1QA) [reply src/util/commonUtils.js]: This issue was fixed

**Pull request merged**